rsts/deployment/cluster_config/auth_setup.rst

```diff
@@ -282,6 +282,7 @@ If your organization does any automated registration, then you'll need to authen
    Flytectl's `config.yaml <https://docs.flyte.org/projects/flytectl/en/stable/#configure>`_ can be
```

Comment:
We can use interSphinx to connect. Let me paste an update here

Reply:
```
:std:ref:`config.yaml <flytectl:flytectl_config>`
```

I think that should do it. Just searched for something that had to do with configuration
```
$ sphobjinv suggest https://flytectl.readthedocs.io config -us

No inventory at provided URL.
Attempting "https://flytectl.readthedocs.io/objects.inv" ...
Remote inventory found.

  Name                                             Score
------------------------------------------------  -------
:std:label:`flytectl_config`                        90
:std:label:`flytectl_config_discover`               90
:std:label:`flytectl_config_init`                   90
:std:label:`flytectl_config_validate`               90
:std:doc:`gen/flytectl_config`                      90
:std:label:`gen/flytectl_config:options`            90
:std:label:`gen/flytectl_config:see also`           90
:std:label:`gen/flytectl_config:synopsis`           90
:std:doc:`gen/flytectl_config_discover`             90
:std:doc:`gen/flytectl_config_init`                 90
:std:label:`gen/flytectl_config_init:options`       90
:std:label:`gen/flytectl_config_init:see also`      90
:std:label:`gen/flytectl_config_init:synopsis`      90
:std:doc:`gen/flytectl_config_validate`             90
:std:label:`index:basic configuration`              90
:std:label:`index:configure`                        90
```

i think this pr will build docs for us, we can check